eloquaStandardAPI/eloquaStandardAPI-connector/eloquaStandardAPI-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/artifacts/ESB/config/proxies/eloquaStandardAPI/eloquaStandardAPI.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <proxy xmlns="http://ws.apache.org/ns/synapse"
```

Comment:
License header needs to be added in the xml configuration (template,proxy service and pom) files as well